cmake/toolchain.linux-x64-asan.cmake

```diff
@@ -0,0 +1,55 @@
+# Toolchain for compiling with ASAN enabled on a Linux-x86-64 host.
+# This is done as a "crosscompile" because we must use our LLVM version
+# of clang for *all* compilation (rather than using it just for bitcode
+# and letting the host compiler, eg gcc, handle everything else); ASAN
+# essentially requires everything to be compiled with matching versions
+# of the same compiler.
+#
+# Note: requires LLVM to be built with -DLLVM_ENABLE_RUNTIMES="compiler-rt;libcxx;libcxxabi;libunwind"
+#
+# Note: only tested with LLVM/Clang 16 as of this comment. Earlier versions
+# may likely work but are not tested.
+
+set(CMAKE_SYSTEM_NAME Linux)
+set(CMAKE_SYSTEM_PROCESSOR i686)
+
+set(LLVM_INSTALL_DIR ${LLVM_DIR}/../../../)
+
+set(CMAKE_C_COMPILER ${LLVM_INSTALL_DIR}/bin/clang)
+set(CMAKE_CXX_COMPILER ${LLVM_INSTALL_DIR}/bin/clang++)
+
+set(_ASAN_FLAGS "-fsanitize=address")
+# set(_ASAN_FLAGS "-fsanitize=address -shared-libasan")
+
+set(CMAKE_CXX_FLAGS_INIT ${_ASAN_FLAGS})
+set(CMAKE_C_FLAGS_INIT ${_ASAN_FLAGS})
+
+set(CMAKE_EXE_LINKER_FLAGS_INIT ${_ASAN_FLAGS})
+set(CMAKE_SHARED_LINKER_FLAGS_INIT ${_ASAN_FLAGS})
+set(CMAKE_MODULE_LINKER_FLAGS_INIT ${_ASAN_FLAGS})
+
+set(CMAKE_FIND_ROOT_PATH_MODE_PROGRAM NEVER)
+set(CMAKE_FIND_ROOT_PATH_MODE_LIBRARY ONLY)
+set(CMAKE_FIND_ROOT_PATH_MODE_INCLUDE ONLY)
+set(CMAKE_FIND_ROOT_PATH_MODE_PACKAGE ONLY)
+
+# Set to empty string to indicate the resulting binaries can be natively executed
+set(CMAKE_CROSSCOMPILING_EMULATOR)
```

Comment:
I think this should be `/usr/bin/env` to be a no-op. Empty makes CMake think "there is no emulator", rather than "no emulator is necessary".

Reply:
Done, but it didn't make any difference.

Reply:
Sorry, but can you show me repro steps that contradict those I gave above?